taskcluster/ci/config.yml

```diff
@@ -0,0 +1,36 @@
+---
+trust-domain: mobile
+treeherder:
+    group-names: {}
+
+task-priority: highest
+
+taskgraph:
+    register: focusios_taskgraph:register
+    repositories:
+        mobile:
+            name: "Focus for iOS"
+    cached-task-prefix: mobile.v2.focus-ios
+
+workers:
+    aliases:
+        # b-linux to be changed to bitrise
```

Comment:
Commenting this since I'm not sure we can already set that to bitrise or we need to have that worker set for focus too before naming it here